c++/src/Compression.cc

```diff
@@ -36,6 +36,10 @@
 #define ZSTD_CLEVEL_DEFAULT 3
 #endif
 
+#ifndef LZ4_ACCELERATION_MAX
+#define LZ4_ACCELERATION_MAX 65537
```

Comment:
Why set to 65537? Will it overflows to be a negative number as the acceleration parameter used in lz4 is of type int.

Reply:
These macros are defined in `lz4.c` instead of the header file for some reason. That's why I needed to redefined them here. Their values are looked up from the docs/[sources](https://github.com/lz4/lz4/blob/bdc9d3b0c10cbf7e1ff40fc6e27dad5f693a00e7/lib/lz4.c#L47-L57).